modules/reindex/src/main/java/org/elasticsearch/index/reindex/TransportReindexAction.java

```diff
@@ -179,16 +186,25 @@ public AsyncIndexBySearchAction(BulkByScrollTask task, ESLogger logger, ParentTa
         @Override
         protected ScrollableHitSource buildScrollableResultSource(BackoffPolicy backoffPolicy) {
             if (mainRequest.getRemoteInfo() != null) {
-                // NORELEASE track 500-level retries that are builtin to the client
```

Comment:
500 error would retry on other nodes, but since we only use a single node this isn't an issue.